### ISSUE TITLE
-Current revision is for Big Sur ? -Question about compatibility with iGPU only config

### ISSUE THREAD

**Issue by klougithub** [SchmockLord/Hackintosh-Intel-i9-10900k-AsRock-Z490-Phantom-ITX-TB3#44]
Dear SchmockLord,
thank you for sharing your config,guide and EFI folder.
I am building an hackintosh that will be 90% equal to your configuration.
I am buying same motherboard, I've the wifi card, and hopefully I'll buy the i9-10900k.
Few questions:
- is this EFI suitable to boot into Big Sur also? currently which bios version you have?
- will it be a problem if I use another intel CPU like i9-10850k or i7-10700k (shouldn't be, just asking if anything should be changed into plist)
- do I need to edit the opencore config if I use only the iGPU? surely I have to use iMac20,2. (right now graphic cards prices are crazy, so I am planning to buy it in the next future)

thanks in advance ;)

**Comment by Hush-vv** [comment]
If you only have a core graphics card, you can set it like this
<img width="793" alt="QQ20210227-122545@2x" src="https://user-images.githubusercontent.com/61744850/109375340-fdc89100-78f6-11eb-8531-93a14b9b9a42.png">
Now I have updated to 11.2.2 (20d80)

**Comment by dreamwhite** [comment]
> will it be a problem if I use another intel CPU like i9-10850k or i7-10700k (shouldn't be, just asking if anything should be changed into plist)

If you have a CPU up to 10700K, you'll have to stay with iMac20,1 SMBIOS, otherwise iMac20,2
Regarding the graphics patch there's no need to add a custom `device-id`, since it's natively supported by macOS Catalina and Big Sur. Same for `framebuffer-unifiedmem`as the BIOS should be already configured with the proper settings.

TL;DR: On the graphics patch just leave `AAPL,ig-platform-id` and delete the other options (test if `enable-hdmi20` is still required)

**Comment by klougithub** [comment]
With the default settings it was breaking HEVC hardware encoding/decoding. (CPU is i9-10900K)
I tried removing one key at time, found out it's one of these 2 that was breaking HEVC hw encoding/decoding:
```
				<key>AAPL,slot-name</key>
				<string>Internal@0,2,0</string>
				<key>device_type</key>
				<string>VGA compatible controller</string>
```

Also I switched to HDMI because it was changing video output after login.
Anyway, What's the purpose of: "hda-gfx=onboard-1" and all _framebuffer_ keys? Are them needed for Audio over HDMI/DP ?

**Comment by ferjums7** [comment]
Anyone can help me. I have the same setup, I am trying to install only with the igpu. When it shows me "about 29 minutes remaining" to finish instalation, the pc restart itself or shutdown. I used the latest EFI folder of schmocklord modyfing the texto you put here showing how to use only the igpu. Nothing works. Help please
